pkg/controllers/rollout/controller_test.go

```diff
@@ -226,6 +227,74 @@ func TestReconcilerUpdateBindings(t *testing.T) {
 		wantErr                    bool
 	}{
 		// TODO: Add negative test cases with fake client
+		"test update bindings with no bindings": {
+			name: "test3",
+			Client: &test.MockClient{
+				MockUpdate: func(ctx context.Context, obj client.Object, opts ...client.UpdateOption) error {
+					return nil
+				},
+			},
+			latestResourceSnapshotName: "snapshot-2",
+			toBeUpgradedBinding:        []*fleetv1beta1.ClusterResourceBinding{},
+			wantErr:                    false,
+		},
+
+		"test update binding with no latestResourceSnapshotName": {
+			name: "Empty latestResourceSnapshotName",
```

Comment:
The name here corresponds to the name of the latest Resource snapshot name it should ideally not follow the test case name

Eg: it could be "testResourceSnapsho1"